data/views/redfish-1.0/redfish.1.0.0.accountservice.1.0.0.json

```diff
@@ -8,13 +8,13 @@
     "Name": "Account Service",
     "Status": {},
     "ServiceEnabled": true,
-    "MinPasswordLength": 8,
-    "MaxPasswordLength": 32,
```

Comment:
Two questions:
(1) What's the doc to describe the requirement on password length? To align the Redfish spec or to match with  the `user` in API 2.0? (https://github.com/RackHD/on-http/blob/master/static/schemas/2.0/user.2.0.json#L15)
(2)  Since ODR-925 has already opened for password length, please also create some unit-test cases to make sure the behavior is expected if password is out-of-range.